### DISCUSSION TITLE
Problem specifying unique credentials in inventory file

### DISCUSSION THREAD

**Discussion by shawn571** [semaphoreui/semaphore#1678] [Q&A]
Probably doing something completely wrong, but I am trying to get a playbook to work that needs to delegate a couple of tasks to a windows host. I've specified these variable in the vars section in a yaml inventory file - ansible_user: winuser@PROD.COMPANY.COM, ansible_password: secretpassword, ansible_connection: winrm and ansible_winrm_transport: kerberos. When I run the playbook it looks like it kind of ignores this because it's using the credential inventorycred which is the SSH credential applied to the inventory. How can I get it to connect to the Windows hosts using the credentials defined in the inventory while still using inventorycred for the rest of the hosts in the inventory? 

1:40:52 PM
fatal: [testwindows]: UNREACHABLE! => {
1:40:52 PM
"changed": false,
1:40:52 PM
"msg": "Kerberos auth failure for principal inventorycred with subprocess: kinit: Client 'inventorycred@PROD.COMPANY.COM' not found in Kerberos database while getting initial credentials",
1:40:52 PM
"unreachable": true
1:40:52 PM
}

**Comment by lbouriez** [comment]
Did you find any solution ?
